### ISSUE TITLE
More robust testing

### ISSUE THREAD

**Issue by njosefbeck** [njosefbeck/gatsby-source-stripe#7]
To test this plugin, I typically use the test data provided by Stripe. I would **love** to test the plugin across more complex, larger Stripe data.

Do you have a Stripe account with a lot of data in it?
Are you a Stripe Connect user?
Are you a part of any closed betas (thinking: issuing)? 

If so, it would be great if you would test this plugin using that data.

**Comment by DavidSabine** [comment]
Hi, I answered 'no' to all three of your question. But I'm really interested in testing this plugin.  My needs are simple:
1) I have 1 product with multiple SKUs (an event that will run on multiple dates)
2) I want to list those items on my site (https://scrum.works/)
3) I want visitors to be able to pay for registration to individual SKUs.

Plugins I except I'll use: gatsby-source-stripe and gatsby-plugin-stripe-elements

My challenge: How to test!???  The e-commerce tutorial at Gatsby/docs is built for gatsby-plugin-stripe-checkout and assumes all kinds of 'magic' (untested/untestable code) in a Lambda function.

I'd like to have a proper set of Jest & Cypress tests.  (that's my definition of "robust testing")  But it's really difficult (for me) to wrap robust tests around the serverless endpoint.  As well, I have questions whether to use Netlify's 'lambda' functions or Gatsby's client-paths for such endpoints.

Basically, I'd love to help.  But I need a lot of help.  ¯\_(ツ)_/¯

**Comment by njosefbeck** [comment]
Closing this for now.
